docs/source/user/notebook.md

```diff
@@ -30,7 +37,7 @@ Drag cells between notebooks to quickly copy content:
 
 [animation]
 
-Create multiple, synchronized views of a single notebook:
+Create multiple synchronized views of a single notebook:
```

Comment:
We should probably describe how to do this using the File menu.

Reply:
I agree